sources/AutoFilerTab.cpp

```diff
@@ -112,17 +119,23 @@ AutoFilerTab::_BuildLayout()
 				.Add(fAutorunBox)
 				.Add(new BSeparatorView(B_VERTICAL))
 				.Add(fStartStop)
-			.End()
+				.End()
 			.Add(new BSeparatorView(B_HORIZONTAL))
 			.Add(folderLabel)
-			.Add(fScrollView)
 			.AddGroup(B_HORIZONTAL)
-				.AddGlue()
-				.Add(fAddButton)
-				.Add(fRemoveButton)
-				.AddGlue()
+				.Add(fScrollView)
+				.AddGroup(B_VERTICAL)
+					.AddGlue()
```

Comment:
Since the buttons on every tab stick to the top, I think they should do here, too, by removing this AddGlue().